turbinia/processors/google_cloud.py

```diff
@@ -34,8 +34,7 @@
 RETRY_MAX = 10
 
 turbinia_nonexisting_disk_path = Gauge(
-    'turbinia_nonexisting__disk_path',
-    'Total number of non existing disk paths')
+    'turbinia_nonexisting_disk_path', 'Total number of non existing disk paths')
```

Comment:
Small thing (sorry I didn't notice this the first time), but we could probably clarify this a bit by adding something like "... after attempts to attach" to the description.

Reply:
fixed